### ISSUE TITLE
 Cannot find package "nuecss"

### ISSUE THREAD

**Issue by langhuihui** [nuejs/www#45]
when i clone this repo and run nue command got error:
```
✓ Initialize ./.dist/dev 
   nuemark.js
   nue.js
   page-router.js
   app-router.js
   mount.js
   diffdom.js
   hotreload.js
   error.css
   favicon.ico

✓ Nue 0.3.1 • Bun 1.0.23 
✓ Serving site from ./.dist/dev
✓ Building site to: ./.dist/dev

✓ Processing styles 42
error: Cannot find package "nuecss" from "/Users/dexter/.bun/install/global/node_modules/nuekit/src/builder.js"
```

**Comment by tipiirai** [comment]
Yeah. The www-repository is currently not compatible with the official version of Nue and has all sorts of inconsistencies in place. I should mention this on the contribution guidelines. Obviously going to convert the repository to be compatible later. But not right now.

**Comment by langhuihui** [comment]
I want to create a docs website，and it's a very good example for reference

**Comment by ArnavK-09** [comment]
> Yeah. The www-repository is currently not compatible with the official version of Nue and has all sorts of inconsistencies in place. I should mention this on the contribution guidelines. Obviously going to convert the repository to be compatible later. But not right now.

hey, can you update this, or public any alpha nuecss version, i want to contribute in this website but this error is not letting me :(

https://github.com/nuejs/nue/blob/638c181a160f1cde4ad4cce82f50666fc1fc71ad/packages/nuekit/src/builder.js#L121

**Comment by nobkd** [comment]
Here's the [current roadmap](https://nuejs.org/#vision):

![Roadmap](https://nuejs.org/img/roadmap4-big.png)

As you can see, `NueCSS` is the next item on the list.
So when he publishes it in [`nuejs/nue`](https://github.com/nuejs/nue), this repo should be updated accordingly.
IN the meantime, we can only contribute by what we can see from the docs' markdown without running it ourselves. :)

**Comment by ArnavK-09** [comment]
> Here's the [current roadmap](https://nuejs.org/#vision):
> 
> ![Roadmap](https://camo.githubusercontent.com/fe321b165226d4e9b72a27178c62a051780ce6cf57e99127d0a8d40a6171e2e5/68747470733a2f2f6e75656a732e6f72672f696d672f726f61646d6170342d6269672e706e67)
> 
> As you can see, `NueCSS` is the next item on the list. So when he publishes it in [`nuejs/nue`](https://github.com/nuejs/nue), this repo should be updated accordingly. 
ohh thanks, would wait till march! 

> IN the meantime, we can only contribute by what we can see from the docs' markdown without running it ourselves. :)
**well already made some**
https://github.com/nuejs/www/pull/47
https://github.com/nuejs/create-nue/pull/36
https://github.com/nuejs/www/pull/48

**Comment by tipiirai** [comment]
I know. This is not a good situation. There is a re-write operation ahead after the Nue CSS is out, since it's not really a pre-processor that is coming, but rather a design system. I want to replace that Stylus-like syntax with Lightning CSS and standards-based CSS (with nesting)

**Comment by ArnavK-09** [comment]
![image](https://github.com/nuejs/www/assets/69188140/992fd1fd-95e7-4a33-af56-724a0f4f6798)


how to enable nuecss
